src/main/java/algoblocks/engine/RepeatBlock.java

```diff
@@ -0,0 +1,31 @@
+package algoblocks.engine;
+
+public class RepeatBlock extends EffectBlock{
+
+  private int times;
+
+  public RepeatBlock(int times){
+    super();
+    this.times = 2;
+    if(times >= 2)
+      this.times = times;
+  }
+
+  public Sequence getSequence() {
+    Sequence sequence = new Sequence();
+    for(int i = 0; i < times; i++){
+      for(Block block: blocks)
+        sequence.concatenate(block.getSequence());
+    }
+    return sequence;
+  }
+
+  public Sequence getInvertedSequence() {
+    Sequence sequence = new Sequence();
+    for(int i = 0; i < times; i++){
+      for(Block block: blocks)
+        sequence.concatenate(block.getInvertedSequence());
+    }
+    return sequence;
```

Comment:
Idem código repetido,
El doble for lo pueden reemplazar por forEach:
```
IntStream.rangeClosed(1, 8)
         .forEach(System.out::println);
```
El ejemplo de arriba imprime del 1 a 8